install/kubernetes/cilium/values.yaml.tmpl

```
@@ -714,6 +714,8 @@ hubble:
     #   --set metrics.enabled="{dns:query;ignoreAAAA,drop,tcp,flow,icmp,http}"
     #
     enabled: ~
+    # -- Enables exporting hubble metrics in OpenMetrics format.
+    enableOpenMetrics: false
```

nit: should we use the same pattern for enabled as for other component in case we need further OpenMetrics specific configuration?
```suggestion
    OpenMetrics:
    # -- Enables exporting hubble metrics in OpenMetrics format.
        enabled: false
```

---

Good question. Currently it's a top-level field on the Prometheus handler, so I was inclined to imagine that it's just a toggle without any additional configuration. We could always add the sub-section later if it's needed, and alias the old `enableOpenMetrics` to `openMetrics.enabled`.